frontend/src/app/services/task.service.ts

```diff
@@ -76,7 +78,13 @@ export class TaskService {
 
     private getTaskFromResponse(response: TaskResponse): Task {
         const tags = response.tags.map(tag => new LabelTag(tag.key, tag.name, tag.tools));
-        return new Task(response.task_id, response.key, response.name, response.image_path, tags,
-            response.datasets_keys);
+
+        // Dev only
+        response.number_of_available_scans = 5;
+        // tslint:disable-next-line: max-line-length
+        response.description = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent scelerisque nulla non laoreet eleifend. Quisque gravida sem sit amet quam consequat malesuada. Nunc tellus nisl, euismod et augue efficitur, egestas sollicitudin neque. Etiam convallis, diam quis convallis posuere, dolor mi blandit ante, ut blandit diam orci quis dolor. Morbi tellus felis, blandit et lorem ac, hendrerit luctus risus. Cras sodales urna ultricies est dignissim tempor. Sed at velit ac odio lacinia dignissim sit amet.';
+        response.label_examples = ['../../../assets/img/login_pic.jpg', '../../../assets/img/login_pic.jpg'];
```

Comment:
Question, do we want this changed? I can add some jpg images representing some random 2 slices.

Reply:
Yep its dev only. There is no need for it since it will be downloaded with task or set to `null` if task dont have description.